threat_model.md

```diff
@@ -0,0 +1,190 @@
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. With *Tanker*, they have no access to Tanker-encrypted data. All future user data will however be encrypted for the attacker's *Tanker* account.
+
+**Identity modification + repeatable data access** :
+
+*Attack*: An attacker gains access to all user data stored by the *application*. They also gain access to all stored *Tanker Identities* and are able to change them. This attack is repeatable. 
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. With *Tanker*, all future user data will be encrypted for the attacker's *Tanker* account. If the attacker accesses the data again in the future, they will gain access to all data since the initial attack.
+
+**App code control**:   
+
+*Attack*: An attacker is able to deploy arbitrary code instead of the *application*. They deploy a version of the *application* which sends all user data in clear to a server they control.
+
+*Impact:* Whatever the level of protection, the attacker has access to all past and future data. 
+
+*Fix:* *Application* verification and signature could help mitigate that risk. This technology however is difficult to implement in browsers.
+
+## User risks factors
+
+| Threat                  | No Tanker                               | Tanker                                  |
+| ----------------------- | --------------------------------------- | --------------------------------------- |
+| Unlocked device         | Targeted access to past and future data | Targeted access to past and future data |
+| Locked device           | Targeted access to past data            | No access                               |
+| Password / Email access | Targeted access to past and future data | Targeted access to past and future data |
+
+### Detailed attacks analysis
+
+**Unlocked device**:   
+
+*Attack*: An attacker gains access to a *user*'s devi*ce. The *user* is logged in the *application*.
+
+*Impact:* The attacker has access to all the *user*'s existing data. If the attacker maintains an open session, they will also have access to future data. 
+
+*Note:* If the *user* revokes the stolen/lost device, the attacker won't have access to future data.
+
+**Locked device**:   
+
+*Attack*: An attacker gains access to a *user*'s *device*. The *user* is not logged in the application.
+
+*Impact:* Depending on the type of *device*, the attacker can have access to part or all of the *user*'s existing data. With Tanker, the attacker won't have access to any data.
+
+**Password / Email access**:
+
+*Attack*: An attacker gains access to a *user*'s password or email address.
+
+*Impact*: As the attacker can use the password or the email to open a session, impact is the same as the *Unlocked device* attack.
```

Comment:
The **Impact** does not match the one from the recap table above. As commented there, there seem to be 2 different cases.